app/models/teacher.rb

```diff
@@ -50,4 +51,15 @@ def display_status
     return "#{SHORT_STATUS[status_before_type_cast]} | #{more_info}" if more_info?
     SHORT_STATUS[status_before_type_cast]
   end
-end
+
+  def display_application_status
```

Comment:
are we doing more db queries than needed? 
can we store the result of db access in a bool, can just check that? Or is rails doing that for us